Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.cpp

```diff
@@ -98,6 +98,23 @@ void RemoteRenderingBackendProxy::ensureGPUProcessConnection()
         });
     }
 }
+template<typename T>
+auto RemoteRenderingBackendProxy::send(T&& message)
+{
+    auto result = streamConnection().send(WTFMove(message), renderingBackendIdentifier(), defaultTimeout);
+    if (UNLIKELY(result != IPC::Error::NoError))
+        RELEASE_LOG(RemoteLayerBuffers, "%p RemoteRenderingBackendProxy::send - failed, name:%" PUBLIC_LOG_STRING ", error:%" PUBLIC_LOG_STRING, this, IPC::description(T::name()), IPC::errorAsString(result));
```

Comment:
Can we log `"[pageProxyID=%" PRIu64 ", webPageID=%" PRIu64 ", renderingBackend=%" PRIu64 "]` like the other calls, so it's easy to correlate with other objects in logging? Might want a shared function for the RELEASE_LOG to make this easier (or macros).